### PR TITLE
FE-02 added delete recipe button with funcionality after click on recipe

### PR DESCRIPTION
**Pull request by kucingz**
https://trello.com/c/AxVmwpq0/2-fe-02-possibility-to-remove-recipe-l

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -18,9 +20,11 @@ import { HttpClientModule } from '@angular/common/http';
   imports: [
     BrowserModule,
     AppRoutingModule,
-    HttpClientModule
+    HttpClientModule,
+    BrowserAnimationsModule,
+    TooltipModule.forRoot()
   ],
-  providers: [],
+  providers: [BsModalService],
```
Comment (Isterius) [inline src/app/app.module.ts:27]: Service should be provided by proper Module. Try importing it instead adding provider (check docs if needed)

**Review by Isterius — CHANGES_REQUESTED** [review]
looks good, but let's try to use Moda Module properly


formatting ignored ;]

**Comment by jakzda** [comment]
zatwierdzam

File: angular.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```
Comment (Isterius) [inline angular.json:34]: why didn't you add this path same as the one below? :D

File: src/styles.scss [hunk 3]
```diff
@@ -5,3 +5,6 @@
 
 /* Importing Datepicker SCSS file. */
 @import "~ngx-bootstrap/datepicker/bs-datepicker";
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```
Comment (Isterius) [inline src/styles.scss:10]: do you know these are global styles, right?

**Review by Isterius — APPROVED** [review]

**Pull request merged**